atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -140,6 +176,10 @@ private AtlasDbHttpClients() {
      */
     private static Client newOkHttpClient(Optional<SSLSocketFactory> sslSocketFactory) {
         com.squareup.okhttp.OkHttpClient client = new com.squareup.okhttp.OkHttpClient();
+
+        // set up cipher suites
```

Comment:
nit: consider eliminating this comment, e.g. by renaming `CONNECTION_SPEC` to `WITH_CYPHER_SUITES` or `SPECS_WITH_CYPHER_SUITES`.